src/Command/GenerateObjectAclCommand.php

```diff
@@ -127,7 +127,7 @@ public function execute(InputInterface $input, OutputInterface $output)
             }
 
             $manipulatorId = sprintf('sonata.admin.manipulator.acl.object.%s', $admin->getManagerType());
-            if ($manipulator = $this->aclObjectManipulators[$manipulatorId] ?? null) {
+            if (!($manipulator = $this->aclObjectManipulators[$manipulatorId] ?? null)) {
```

Comment:
```suggestion
            $manipulator = $this->aclObjectManipulators[$manipulatorId];
            
            if (null === $manipulator) {
```

Reply:
Is `$this->aclObjectManipulators[$manipulatorId]` always defined? Otherwise, the assignment will kept as it was (using the `??` operator): `$manipulator = $this->aclObjectManipulators[$manipulatorId] ?? null`.

Reply:
I didn't write this code, it was recently introduced and I just added a negation.

Reply:
E.G. the check passed if aclObjectManipulators is an empty array. And the key provided was `sonata.admin.manipulator.acl.object.orm`. I just made sure, that it doesn't continue afterwards.